src/main/features/response/routes/task-list.ts

```diff
@@ -28,7 +28,8 @@ export default express.Router()
           submitSection: submitSection,
           respondToClaimSection: respondToClaimSection,
           claim: claim,
-          isAfter4pm: isAfter4pm()
+          isAfter4pm: isAfter4pm(),
+          paths: Paths
```

Comment:
preferred approach is to add these as nunjucks globals see nunjucks/index.ts

Reply:
done